NativeApp/src/components/Login/Input/index.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import { StyleSheet } from 'react-native';
+import { FormInput } from 'react-native-elements';
+import { PropTypes as PT } from 'prop-types';
+import { UNOBLUE, BLACK, RED, GREY } from '../../../styles/colors';
+
+class Input extends React.Component {
+  static propTypes = {
+    value: PT.node.isRequired,
+    textInputRef: PT.func,
+    onChangeText: PT.func.isRequired,
+    returnKeyType: PT.string.isRequired,
+    onBlur: PT.func,
+    onFocus: PT.func,
+    hasError: PT.bool,
+    placeholder: PT.string.isRequired,
+    onSubmitEditing: PT.func.isRequired,
+    blurOnSubmit: PT.bool,
+    secureTextEntry: PT.bool,
+  };
+
+  static defaultProps = {
+    onFocus: () => {},
+    onBlur: () => {},
+  }
+
+  state = {
+    isFocused: false,
+  };
+
+  onFocus = (e) => {
+    const {
+      onFocus,
+    } = this.props;
```

Comment:
Your could have this on a single line. It's only when it gets really long I would split it up.

Reply:
yea, i agree :)